packages/core/src/TagsInput/TagsInput.js

```diff
@@ -239,6 +239,7 @@ const HvTagsInput = (props) => {
       const newTagsArr = [...value.slice(0, i), ...value.slice(i + 1)];
       setValidationState(validationStates.standBy);
       setValue(newTagsArr);
+      setResetInput(false);
       setTagCursorPos(newTagsArr.length);
       inputRef.current?.focus();
       performValidation(newTagsArr);
```

Comment:
Not related with this change: I notice the delete code is almost triplicated... Couldn't it be abstracted in some way and be called at the three delete methods?